diff --git a/crying_counter/main.py b/crying_counter/main.py
index 7af3068..3067fc5 100644
--- a/crying_counter/main.py
+++ b/crying_counter/main.py
@@ -4,8 +4,8 @@ import discord
 from discord.ext import commands
 
 
-from matcher import is_similar
-from db.database import Database
+from .matcher import is_similar
+from .db.database import Database
 from settings import BOT_TOKEN
 
 intents = discord.Intents.all()

